flight/views.py

```diff
@@ -1,3 +1,17 @@
-from django.shortcuts import render
+import json
+from django.views  import View
+from django.http   import JsonResponse
+from flight.models import Flight
 
-# Create your views here.
+class FlightImageView(View):
+    def get(self, request):
+        flights = Flight.objects.select_related('airport_depart', 'airport_arrive', 'image', 'calender_price').all()
+        flight  = [{
+            'id'     : flight.id,
+            'date'   : flight.depart_time,
+            'price'  : flight.calender_price.price.price,
+            'depart' : flight.airport_depart.korean_name,
+            'arrive' : flight.airport_arrive.korean_name,
+            'image'  : flight.image.image_url
```

Comment:
```suggestion
                'id'     : flight.id,
                'date'   : flight.depart_time,
                'price'  : flight.calender_price.price.price,
                'depart' : flight.airport_depart.korean_name,
                'arrive' : flight.airport_arrive.korean_name,
                'image'  : flight.image.image_url
```
select_related good! 참조 관계 깔끔하네요 ! :0

Reply:
수정하였습니다!